### DISCUSSION TITLE
Azure Policy implementation options

### DISCUSSION THREAD

**Discussion by rashabiharip** [aztfmod/terraform-azurerm-caf#1331] [Q&A]
How the governance policies (Azure policy definition, assignment, etc.) are managed using this caf module ?

**Comment by rashabiharip** [comment]
Dear @LaurentLesle @arnaudlh , just wanted to confirm if any governance policies implementation we have here.. Thanks in advance.

**Reply by bkarstaedt** [reply]
Hi @rashabiharip, you probably might want to have a look at this `terraform-azurerm-caf-enterprise-scale` article: https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/blob/246cf7de44369b2a40a1b07fafdd8b8b792e7c35/docs/wiki/%5BExamples%5D-Create-Custom-Policies-Policy-Sets-and-Assignments.md

**Reply by arnaudlh** [reply]
hi @rashabiharip, @bkarstaedt is correct, we dont manage those components as our sister team does a fantastic job in it!

**Reply by rashabiharip** [reply]
Thank you @bkarstaedt & @arnaudlh for your guidance, checking if we have any reference implementation/example of this ES-ALZ here in CAF-LZ.
